src/cohere/client.py

```diff
@@ -121,6 +124,19 @@ def __init__(
     delete_connector: Never = moved_function("delete_connector", ".connectors.delete")
     oauth_authorize_connector: Never = moved_function("oauth_authorize_connector", ".connectors.o_auth_authorize")
 
+    def local_tokenize(self, *, text: str, model: str) -> TokenizeResponse:
+        from .manually_maintained.tokenizers import local_tokenize
+        # TODO: parse into TokenizeResponse()? or should remain decoupled from the API responses?
```

Comment:
yeah i think so if it fits neatly into that type

Reply:
After a deep dive .. the answer is no. 
In tokenize endpoint, we return the textual representation of **each** token, which is not achievable unless we called `decode()` on the encoded token ids. Since the local tokenization flow is supposed to optimize for speed, this might not seem like a good idea.